apps/user/templates/channels.jade

```diff
@@ -3,11 +3,10 @@ extends ../../../components/layout/index
 block body
   include _path
   .UserChannels.js-user-channels
-    if user && user.isPremium()
-      .flex-container
-        - searchee = (user && author.id == user.id) ? "your " : author.get('username') + "'s public "
-        - title = searchee + "channels"
-        include ../../../components/filter/templates/search_bar
+    .flex-container
+      - searchee = (user && author.id == user.id) ? "your " : author.get('username') + "'s public "
+      - title = searchee + "channels"
```

Comment:
searcheeeeeee

Reply:
searché

Reply:
One who is being searched on is a searchee

Reply:
s̩̀̏̒̇e̘͇͎ͯ̂̋ͅă̜̣͆̐ͥr̴͔̝͖͎̂͆c̆̿ͩ̚h̫͓̦̾͒̍͌̃e̝͖e̙̔ͦ͗̋̍͆̐